spec/inputs/heartbeat_spec.rb

```diff
@@ -1 +1,92 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/heartbeat"
+
+def fetch_event(config, number)
+  pipeline = LogStash::Pipeline.new(config)
+  queue = Queue.new
+  pipeline.instance_eval do
+    @output_func = lambda { |event| queue << event }
+  end
+  pipeline_thread = Thread.new { pipeline.run }
+  counter = 0
+  begin
+    event = queue.pop
+    pipeline_thread.join
+    counter += 1
+  end until counter == number
+  return event
+end
+
+describe LogStash::Inputs::Heartbeat do
+  context "Default message test" do
+    subject do
+      next LogStash::Inputs::Heartbeat.new({})
+    end
+
+    it "should generate an 'ok' message" do
+      sequence = 1
+      subject.generate_message(sequence) do |event|
+        insist { event['message'] } == 'ok'
+      end
```

Comment:
We are deprecating the insist gem in logstash and moving to use rspec3 only,
The correct form for the tests would be:
```ruby
expect(event['message']).to eq('ok')
```